arch/x86/core/intel64/thread.c

```diff
@@ -73,3 +77,28 @@ int arch_float_enable(struct k_thread *thread, unsigned int options)
 	/* x86-64 always has FP/SSE enabled so nothing to do here */
 	return 0;
 }
```

Comment:
This is also unnecessary given the other changes here. It's also not sufficient as it doesn't cover the SMP case as those cores won't go through this path and so won't have their FS_BASE MSR set.